Zend/zend_compile.c

```diff
@@ -131,7 +131,7 @@ static zend_string *zend_build_runtime_definition_key(zend_string *name, unsigne
 {
 	zend_string *result;
 	char char_pos_buf[32];
-	size_t char_pos_len = zend_sprintf(char_pos_buf, "%p", lex_pos);
+	size_t char_pos_len = sprintf(char_pos_buf, "%p", lex_pos);
```

Comment:
Not directly related, but this whole function should be ported to use zend_strpprintf.

Reply:
I'll check this one and send a separate pull request for this...